src/Compilers/CSharp/Portable/Symbols/ReducedExtensionMethodSymbol.cs

```diff
@@ -625,6 +625,8 @@ public override ImmutableArray<CustomModifier> RefCustomModifiers
                 }
             }
 
+            public override Symbol AssociatedSymbol => null;
```

Comment:
for many cases, the `=> null` impl was simple and obvious.  for example, as this is only for record-primary-cosntructor, there is no way this returns anything for a reduced extensionmethod parameter.  HOwever, some  later cases are less clear to me.